### PR TITLE
Use Python 3.9 and Bullseye dependencies

### PR DESCRIPTION
**Pull request by sopelj**

### CODE REVIEW

**Review by fblackburn1 — APPROVED** [review]

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
 https://github.com/wazo-platform/xivo-manage-db/archive/master.zip
-dbus==1.2.8
+dbus-python==1.2.16
```
Comment (fblackburn1) [inline requirements.txt:2]: it seems to cause issue

Comment (sopelj) [reply requirements.txt:2]: Yeah, it only causes issues now, because the other was never tested. There is no package `dbus`.

Comment (sopelj) [reply requirements.txt:2]: It installs fine locally. It must be missing something on the CI... Looks like it requires dbus to build.

Comment (sopelj) [reply requirements.txt:2]: Should I remove the test for installation (it was not there before) or install the dependency in the docker? A shame it won't resolve on its own.

**Review by fblackburn1 — APPROVED** [review]

**Pull request merged**